Resources/LaboratoryExchange-DiagnosticReport.xml

```diff
@@ -0,0 +1,250 @@
+      <binding>
+        <strength value="required" />
+        <description value="Code for Diagnostic Service Codes" />
+      </binding>
+    </element>
+    <element id="DiagnosticReport.category:resultType">
+      <path value="DiagnosticReport.category" />
+      <sliceName value="resultType" />
+      <short value="ResultType" />
+      <definition value="The type of result defines the laboratory specialty under which the test is categorized." />
+      <alias value="ResultaatType" />
+      <binding>
+        <strength value="required" />
+        <description value="Codes for high level observation categories." />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="lu-dataset-2" />
+        <map value="lu-concept-v2-4289" />
+        <comment value="ResultType" />
+      </mapping>
+    </element>
+    <element id="DiagnosticReport.code">
+      <path value="DiagnosticReport.code" />
+      <short value="PanelOrBattery" />
+      <definition value="For laboratory tests comprising multiple subtests and often requested together as a whole, this concept contains the name of the compound request (often indicated as a ‘panel’, ‘battery’ or ‘cluster’). Examples include: blood gases and EBV serology." />
+      <alias value="Onderzoek" />
+      <mustSupport value="true" />
+      <binding>
+        <strength value="extensible" />
+        <description value="PanelOrBatteryCodelist" />
+        <valueSet value="https://decor.nictiz.nl/ad/#/lu-/terminology/valueset/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.5" />
+      </binding>
+      <mapping>
+        <identity value="lu-dataset-2" />
+        <map value="lu-concept-v2-4286" />
+        <comment value="PanelOrBattery" />
+      </mapping>
+    </element>
+    <element id="DiagnosticReport.subject">
+      <path value="DiagnosticReport.subject" />
+      <short value="Patient" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+      <mustSupport value="true" />
+      <mapping>
+        <identity value="lu-dataset-2" />
+        <map value="lu-concept-v2-4303" />
+        <comment value="Patient" />
+      </mapping>
+    </element>
+    <element id="DiagnosticReport.issued">
+      <path value="DiagnosticReport.issued" />
+      <mustSupport value="true" />
+    </element>
+    <element id="DiagnosticReport.performer">
+      <path value="DiagnosticReport.performer" />
+      <short value="Performer" />
+      <definition value="The healthcare provider and/or organization where or by whom the LaboratoryTestResult was performed. This is not necessarily the actual performer of (all) laboratory tests. Specialization: because of: Definition." />
+      <alias value="Uitvoerder" />
+      <type>
+        <code value="Reference" />

```

Comment:
The wording of the comment suggests that this can be a place or an organization, but only the Organization resource is allowed. I think this needs to be tweaked a bit.

In addition, what does "Specialization: because of: Definition." mean?